src/benchmarks/micro/libraries/System.IO.Compression/Brotli.cs

```diff
@@ -13,7 +13,7 @@ public class Brotli : CompressionStreamPerfTestBase
         private const int Window = 22;
 
         public override Stream CreateStream(Stream stream, CompressionMode mode) => new BrotliStream(stream, mode);
-        public override Stream CreateStream(Stream stream, CompressionLevel level) => new BrotliStream(stream, level);
+        public override Stream CreateStream(Stream stream, CompressionLevel level) => new BrotliStream(stream, level, leaveOpen: true);
```

Comment:
The same thing needs to be done in these:
https://github.com/dotnet/performance/blob/ccbeb4ed946e0a7bb676c58f6b5841740c616873/src/benchmarks/micro/libraries/System.IO.Compression/CompressionStreamPerfTestBase.cs#L11-L28

I think it'd also be cleaner if both CreateStream overloads always set leaveOpen: true, so it doesn't differ by which is invoked in the tests.